### ISSUE TITLE
One-Click Deploy not working on Ghost 5

### ISSUE THREAD

**Issue by sanjaydotpro** [render-examples/ghost#11]
Due to the recent upgrade to Ghost 5, the "One-Click Deploy" to Render is not working anymore.

**Comment by vanjacosic** [comment]
I submitted PR #12 recently, hopefully it's a step towards fixing this issue once someone has a chance to review it 🤞

**Comment by sanjaydotpro** [comment]
Yes, it works after a few tweaks but I am not sure how to add a default domain name for the ghost blog. After deploying, for me it assigned a domain automatically as "https://ghost-pjy2.onrender.com/" and I can't remove it now even if I add a custom domain. I want the domain to be "blog.payqual.app". Is there any way I can update the URL in ghost?

**Comment by zach-render** [comment]
With the update to Ghost 5 requiring using MySQL as the production database, we've made the decision to remove the 1-click Deploy to Render button. We've updated the [Ghost Quickstart](https://render.com/docs/deploy-ghost) to reflect the new deployment process. Thus, this issue will be closed with a `wontfix` label.

As for running your Ghost instance behind a custom URL, you'll want to follow the steps outlined in the [Ghost Quickstart](https://render.com/docs/deploy-ghost#custom-domains-for-ghost). That means that in addition to setting DNS records at your as appropriate, you'll also need to add an environment variable to your Ghost instance as documented. Doing both of these steps will allow your Ghost instance to be available at your custom domain.
